lib/parser/heading.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Parser
+  class Heading
```

Comment:
I think that it makes more sense to have `Parser::GithubToJira::Heading`, `Parser::JiraToGithub::Heading` than these two methods in a single class. We don't share code between the implementations and at the point of calling the code we always know in which direction we want to perform the translation.

Reply:
Agree 💯